content/sensu-go/5.12/reference/backend.md

```diff
@@ -25,6 +25,7 @@ menu:
   - [Dashboard configuration](#dashboard-configuration-flags)
   - [Datastore and cluster configuration](#datastore-and-cluster-configuration-flags)
   - [Advanced configuration options](#advanced-configuration-options)
+  - [Configuration via environment variables](#configuration-via-environment-variables)
```

Comment:
I believe that these env vars will be supported in the backend starting with the 5.15 release, so let's move this change into that version of the content. It should not be present in versions prior to 5.15.